packages/cli/lib/sandboxes.js

```diff
@@ -1,73 +1,66 @@
-const chalk = require('chalk');
 const { i18n, MISSING_LANGUAGE_DATA_PREFIX } = require('./lang');
 const { handleExit, handleKeypress } = require('./process');
 const { logger } = require('@hubspot/cli-lib/logger');
-const {
-  getAccessToken,
-  updateConfigWithAccessToken,
-} = require('@hubspot/local-dev-lib/personalAccessKey');
 const { EXIT_CODES } = require('./enums/exitCodes');
-const { enterAccountNamePrompt } = require('./prompts/enterAccountNamePrompt');
 const {
   fetchTaskStatus,
   fetchTypes,
   getSandboxUsageLimits,
 } = require('@hubspot/local-dev-lib/sandboxes');
-const {
-  accountNameExistsInConfig,
-  getConfig,
-  writeConfig,
-  updateAccountConfig,
-  getAccountId,
-} = require('@hubspot/local-dev-lib/config');
+const { getConfig, getAccountId } = require('@hubspot/local-dev-lib/config');
 const CliProgressMultibarManager = require('./CliProgressMultibarManager');
 const { promptUser } = require('./prompts/promptUtils');
 const { getHubSpotWebsiteOrigin } = require('@hubspot/local-dev-lib/urls');
 const {
-  personalAccessKeyPrompt,
-} = require('./prompts/personalAccessKeyPrompt');
-const { logErrorInstance } = require('./errorHandlers/standardErrors');
-
-const STANDARD_SANDBOX = 'standard';
-const DEVELOPER_SANDBOX = 'developer';
+  HUBSPOT_ACCOUNT_TYPES,
+} = require('@hubspot/local-dev-lib/constants/config');
+const {
+  STANDARD_SANDBOX_TYPE,
+  DEVELOPER_SANDBOX_TYPE,
+  DEVELOPER_SANDBOX,
+  STANDARD_SANDBOX,
+} = require('./constants');
 
 const syncTypes = {
   OBJECT_RECORDS: 'object-records',
 };
 
 const sandboxTypeMap = {
```

Comment:
Do we need this just because of how `sandboxAccountType` works? Seems like we can probably also remove this once we're ready to fully remove `sandboxAccountType`, right?

Reply:
yeah good point... its a bit confusing cause our own sandboxes endpoints use `DEVELOPER` and `STANDARD` for sandbox types which is why I had kept them, but I'll revisit all these once I get to the `sandboxAccountType` migration issue